lib/Cake/View/Helper/FormHelper.php

```diff
@@ -435,7 +441,13 @@ public function create($model = null, $options = array()) {
 		}
 		$this->requestType = strtolower($options['type']);
 
-		$action = $this->url($options['action']);
+		if ($options['action'] === false || $options['url'] === false) {
+			$action = null;
+		} else {
```

Comment:
Could this else be removed by setting action = null before the condition and inverting the condition?